Assets/BossRoom/Scripts/Shared/Game/UI/UIHealth.cs

```diff
@@ -20,9 +20,7 @@ public void Initialize(NetworkVariableInt networkedHealth, int maxValue)
 
             m_HitPointsSlider.minValue = 0;
             m_HitPointsSlider.maxValue = maxValue;
-            m_HitPointsSlider.value = networkedHealth.Value;
-            // disable slider when we're at full health!
```

Comment:
Add comment here inside of HealthChanged (so we're aware going forward of this design change).

Reply:
ok

Reply:
Looks good to me. I'll create a Jira task for not reviving players at max health (we might already have one) since I would want to see my health bar once I'm revived. So setting a value of 50% of max health for example would be a good design solution to this.